### ISSUE TITLE
audio is not working

### ISSUE THREAD

**Issue by sathishvudugundla** [OpenVidu/openvidu#595]
when two participants are already in the call when third person is joined from ios mobile , the remaining two persons will able to hear the audio of third person but third person in not able to hear the voice of other two persons. but one to one call is working perfectly any one please help me

**Comment by j1elo** [comment]
> but one to one call is working perfectly

Do you mean that one-to-one works perfectly when one is using iOS?
What happens if the two people use iOS (in the one-to-one scenario)?

* OpenVidu version?
* CE or PRO?
* third person joins using app (ionic)? or Safari browser?

**Comment by sathishvudugundla** [comment]
yes one to one works fine in any case like two ios or one ios, one android or  one web and  one ios
openvidu version is v2.15.0
i am using iphone SE
this issue occured only third person joins from ios(swift)

**Comment by j1elo** [comment]
Please stop all traffic, then change the Kurento debug level to this value (note it is a very long single line):

`3,Kurento*:4,kms*:4,sdp*:4,webrtc*:4,*rtpendpoint:4,rtp*handler:4,rtpsynchronizer:4,agnosticbin:4,KurentoMediaElementImpl:5,agnosticbin*:5,kmssdpsession:5,webrtcendpoint:5,kmswebrtcsession:5,kmsiceniceagent:5`

Instructions to change the Kurento debug level are included in next message:

**Comment by j1elo** [comment]
OpenVidu 2.15 CE
================

Change Kurento Log Level
------------------------

* Method: Edit the configuration file.
* Doc: [Change log level of the services](https://docs.openvidu.io/en/2.15.0/deployment/deploying-on-premises/#change-log-level-of-the-services).

```
ssh OPENVIDU_IP
cd /opt/openvidu/
editor .env
```

Edit or add the property `KMS_DEBUG_LEVEL` with the desired value. You can see examples of commonly used values, in [Kurento Debug Logging](https://doc-kurento.readthedocs.io/en/latest/features/logging.html#suggested-levels).

After editing the `.env` file, restart OpenVidu:

```
./openvidu restart
```



Get Kurento Logs
----------------

* Method: Get the logs locally.
* Doc: [Show service logs](https://docs.openvidu.io/en/2.15.0/deployment/deploying-on-premises/#show-service-logs).

```
ssh OPENVIDU_IP
cd /opt/openvidu/
docker-compose logs kms
```



OpenVidu 2.15 PRO
=================

Change Kurento Log Level
------------------------

* Method: Edit the configuration file.
* Doc: [Change log level of the services](https://docs.openvidu.io/en/2.15.0/openvidu-pro/deployment/on-premises/#change-log-level-of-the-services_1).

Repeat this for each of the Media Node(s):

```
ssh MEDIA_NODE_IP
cd /opt/kms/
editor .env
```

Edit or add the properties `KMS_DEBUG_LEVEL` and `GST_DEBUG` with the desired value. You can see examples of commonly used values, in [Kurento Debug Logging](https://doc-kurento.readthedocs.io/en/latest/features/logging.html#suggested-levels).

Note that **both properties must exist and have the same value** (this requirement will be removed in OpenVidu 2.16).

After editing each `.env` file, restart the Media Node:

```
./media_node restart
```

After restarting all Media Node(s), restart OpenVidu:

```
ssh OPENVIDU_SERVER_PRO_IP
cd /opt/openvidu/
./openvidu restart
```



Get Kurento Logs
----------------

* Method 1: Get the logs locally.
* Doc: [Troubleshooting Media Nodes](https://docs.openvidu.io/en/2.15.0/openvidu-pro/deployment/on-premises/#troubleshooting-media-nodes).

Repeat this for each of the Media Node(s):

```
ssh MEDIA_NODE_IP
cd /opt/kms/
docker-compose logs kms
```

* Method 2: Use Kibana to query Kurento logs.
* Doc:
  - [Monitoring](https://docs.openvidu.io/en/2.15.0/openvidu-pro/detailed-session-monitoring/).
  - [Reviewing logs](https://docs.openvidu.io/en/2.15.0/openvidu-pro/detailed-session-monitoring/#reviewing-logs).
  - [Searching Examples](https://docs.openvidu.io/en/2.15.0/openvidu-pro/detailed-session-monitoring/#searching-examples).
* Steps:
  1. Open the Kibana dashboard which comes included with OpenVidu PRO:
     `https://DOMAIN_OR_PUBLIC_IP/kibana/`
  2. Search with the index pattern `filebeat-kurento*`

**Comment by j1elo** [comment]
When the new log level has been configured, perform a test where the connection fails. Then, send us the logs to see if there are any WARNING or ERROR message that helps to see what went wrong.
